items/boosters.json

```diff
@@ -2,81 +2,41 @@
 	"684729784": {
 		"name": "Stamina Enhancement",
 		"description": "Increases all Helldivers' stamina capacity and recovery.",
-		"unlock": "Helldivers Mobilize Warbond",
-		"cost": {
-			"individual": 75,
-			"total": 495,
-			"page": 7
-		}
+		"unlock": "Helldivers Mobilize Warbond"
```

Comment:
I should've been more specific, `unlock` falls in the same category as `cost`
AFAIK this is already contained in the warbond JSON files?